src/pages/tBTC/Bridge/components/DepositDetailsStep.tsx

```diff
@@ -42,8 +22,8 @@ const BitcoinConfirmationsSummary: FC<{
           {"/"}
           {minConfirmationsNeeded}
         </Skeleton>
-        {"  Bitcoin Network Confirmations"}
-      </BodySm>
+        &nbsp;Bitcoin Network Confirmations
```

Comment:
Let's remove `$nbsp;` here and replace it with `{" "}` (or with the previous implementation). We don't want non-breaking space here.

Reply:
Ref commit: b4762d42af63550fa0e59c31bf3bc7ef52ec7844